src/EvoSC.Common/Config/ConfigServiceExtensions.cs

```diff
@@ -11,6 +11,12 @@ public static class ConfigServiceExtensions
 {
     private const string MainConfigFile = "config/main.toml";
     
+    /// <summary>
+    /// Set up and parse EvoSC's base configuration. This method also adds the configuration
+    /// to the service container.
+    /// </summary>
+    /// <param name="services"></param>
```

Comment:
Could use a quick and short description of services.

Reply:
This is an extension method, you can only call it on an object you already know about and what it is. I think documentation for the parameter that is extended is not neccessary.